.github/workflows/update-test-results.yml

```
@@ -54,15 +54,19 @@ jobs:
             echo '## Summary'
             echo
             echo '| Container | Average | Minimum | Maximum |'
-            echo '| --- | --- | --- | --- |'
-            printf "%b" "$summary"
-          } > README.md
+          echo '| --- | --- | --- | --- |'
+          printf "%b" "$summary"
+        } > README.md
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 59: did not find expected comment or line break

(syntax-check)